ContactDetailsApi.Tests/MockWebApplicationFactory.cs

```diff
@@ -38,6 +38,7 @@ public class MockWebApplicationFactory<TStartup>
 
         public MockWebApplicationFactory()
         {
+            //Environment.SetEnvironmentVariable("")
```

Comment:
Can probably be removed?
If anything, the `EnsureEnvVarConfigured` could be used instead should we need to set any more vars.